python/lsst/sconsUtils/tests.py

```diff
@@ -191,3 +195,68 @@ def run(self, fileGlob):
             self._env.Clean(target, self._tmpDir)
 
         return targets
+
+    def runPythonTests(self, pyList):
+        """Add a single target for testing all python files. pyList is
+        a list of nodes corresponding to python test files. The
+        IgnoreList is respected when scanning for entries. If pyList
+        is None, we will use automated test discovery within pytest.
```

Comment:
The condition for automatic test discovery is a bit more complicated than that. It is, at least: `If pyList is None or an empty list...` but if no tests are found in the provided nodes then it also goes into auto discovery mode, and that worries me a bit because auto discovery may be triggered unexpectedly.